client/src/pages/Home.js

```diff
@@ -1,11 +1,27 @@
 import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import MainTable from "../Component/MainTable";
+import SMSModal from "../Component/SMSModal";
 
 import "./Home.css";
 
+
+
 export function Home() {
 	const [message, setMessage] = useState("Loading...");
+	const [checkedCheckboxes, setCheckedCheckboxes] = useState([]);
+
+	const onHandleSelectedUserState = (isSelected, client_id ) => {
```

Comment:
```suggestion
	const onHandleSelectedUserState = (isSelected, clientId ) => {
```
Nitpick: Let's update the client id variable to camelCase. I mentioned `client_id` in my other comment because it's currently used in `MainTable.js` with an underscore, but that's the only other reference of client id in the codebase. `client_id` is how we're getting it from the backend so in `MainTable.js` we have to use it with an underscore. Here, we're not forced to use underscore so let's use the JS convention of camelCase. There might be a way to get it as camelCase from the backend for better consistency but we'd have to investigate that first (for another time)